src/store/RelayContext.js

```diff
@@ -6,7 +6,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
```

Comment:
Let's keep `RelayStore.js` as a module; it should export a default instance of `RelayContext`. We don't need a test for this, only a mock